build/build.xml

```diff
@@ -352,7 +361,12 @@
   <target name="solr:war" depends="set-classpath, bootstrap-downloads" description="Repack the Solr .war file with our configs">
     <copy file="${solr_file}" tofile="solr.war" overwrite="true" />
     <war destfile="solr.war" update="true">
-      <zipfileset dir="../solr" prefix="WEB-INF/classes"/>
+      	    <zipfileset dir="../solr" includes="schema.xml" prefix="WEB-INF/classes"/>
+	    <zipfileset dir="../solr" includes="solrconfig.xml" prefix="WEB-INF/classes"/>
+	    <zipfileset dir="../solr" includes="stopwords.txt" prefix="WEB-INF/classes"/>
+	    <zipfileset dir="../solr" includes="synonyms.txt" prefix="WEB-INF/classes"/>
+   	    <zipfileset dir="../solr" includes="Dockerfile" prefix="WEB-INF/classes"/>
```

Comment:
@michael-lts a small thing, the Dockerfile shouldn't be bundled with the war.

Reply:
Thank you @mark-cooper and @lorawoodford . We are still working on bringing up a shared test instance, and I will keep you posted on that qa testing. Thanks for letting me know about the eval plan, sounds good. There are additional solr fields in other pull requests, I will let you know about those soon. Your point about pulling the jars in bootstrap is a good one, I should have thought of it! And thank you for pointing out about not bundling the Dockerfile in the war.

Reply:
@mark-cooper, @lorawoodford, regarding new schema fields:
The additional new solr fields are in the schema on our developement branch here:
https://github.com/harvard-library/archivesspace/blob/development/solr/schema.xml
They are:
_u_abool (line 260)
_u_typeahead_utext (line 263)
_u_typeahead_usort (line 264 - uses new icu_sort)

Reply:
@mark-cooper and @lorawoodford , I made the suggested changes:
Added the 3 solr sort-related jars to bootstrap and removed from source
Added solr/lib jars to gitignore
Removed Dockerfile from solr:war target
I also added solr/lib/.gitkeep to initialize that directory for download
Thanks!